t/data/tests/tests/boot.pm

```diff
@@ -50,6 +50,10 @@ END
     script_run "echo '924095f2cb4d622a8796de66a5e0a44a  text' > text.md5";
     assert_script_run 'md5sum -c text.md5';
 
+    # TinyCore busybox sh acts as bash but does not provide it so we do here
+    script_run 'alias bash=sh', 0;
```

Comment:
@coolo this makes it work for tinycore. Unfortunately I can't add a test for downloading as we have no curl in tinycore